### PR TITLE
Adding a currency formatting component

### PR DESCRIPTION
**Pull request by sri-ds**
Keep relevant lines. You should select one choice for each category
(e.g. **Extern**), if none of choices is valid you should probably do
something or at least add a comment here.

New package: react-number-format

**Extern:** Written by hand.

Update:

### CODE REVIEW

**Comment by martinklepsch** [comment]
Closure library also provides `goog.i18n.NumberFormat` which probably does similar things like this: https://google.github.io/closure-library/api/goog.i18n.NumberFormat.html

Doesn't hurt to merge new packages like this but generally using Closure library has some nice benefits (dead code elimination, no dependency).

File: react-number-format/build.boot [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(require '[boot.core :as c]
+         '[boot.tmpdir :as tmpd]
+         '[clojure.java.io :as io]
+         '[clojure.string :as string])
+
+(deftask package []
+  (comp
+   (download :url (str "https://github.com/s-yadav/react-number-format/archive/v2.0.0-alpha2.zip")
```
Comment (Deraen) [inline react-number-format/build.boot]: Use the `+lib-version+` var you have defined to build the url:

```clj
(def +lib-version+ "1.2.3")

...

(download :url (format "https://github.com/example/project/archive/%s.zip" +lib-version+)
          :unzip true)
```

File: react-number-format/build.boot [hunk 2]
```diff
@@ -0,0 +1,38 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]
+                  [cljsjs/react "15.3.1-0"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "2.0.0")
```
Comment (Deraen) [inline react-number-format/build.boot]: Use "2.0.0-alpha2" as the lib version and append just -0 as the build identifier to version.

File: react-number-format/build.boot (hunk 1 above)
Comment (sri-ds) [inline react-number-format/build.boot]: Just now made the changes you requested.

Comment (sri-ds) [inline react-number-format/build.boot]: @martinklepsch This is a text input component that allows for formatting currency as the user  enters in the text field.

Comment (sri-ds) [inline react-number-format/build.boot]: @Deraen Have made the changes you requested. Can you look at this pr again to see if this can be merged? Thanks.

**Pull request merged**